docs/architecture.md

```diff
@@ -0,0 +1,147 @@
+# Peer Pods Architecture
+
+## Overview
+
+The cloud-api-adaptor is an implementation of the
+[remote hypervisor interface](https://github.com/kata-containers/kata-containers/blob/CCv0/src/runtime/virtcontainers/remote.go)
+of [Kata Containers](https://github.com/kata-containers/kata-containers)
+
+It enables the creation of Kata Containers VMs on any machines without the need for bare metal worker nodes,
+or nested virtualisation support.
+
+## Background
```

Comment:
This background text is really good; it would fit very well on the CoCo's main docs indeed.